### PR TITLE
initial nest conda build files

### PR DESCRIPTION
**Pull request by planetmija**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/nest/build.sh [hunk 1]
```diff
@@ -21,3 +21,8 @@ make -j2
 make install
 # this is needed to make the python bindings work
 cp $PREFIX/lib64/* $PREFIX/lib -r
+
+extra:
+  container:
+	    # openmpi needs ssh/rsh
+	    extended-base: true
```
Comment (mbargull) [inline recipes/nest/build.sh]: `build.sh` -> `meta.yaml` 😉

**Comment by bgruening** [comment]
@bioconda/core this is our first recipe with openmpi. Openmpi needs openssh installed, which is really tricky.
This recipe builds with with following patches for our build-infrastructure:
* https://github.com/bioconda/bioconda-utils/pull/211
* https://github.com/bioconda/bioconda-extended-base-image/pull/6

This work was done and discussed during our workshop yesterday with admins from out local compute center - (HPC and Cloud).

**Comment by johanneskoester** [comment]
Sounds like a reasonable strategy to me. I guess openssh as a conda package is not feasible?

**Comment by daler** [comment]
The conda package for `nest` will still require a system installation of openssh though, right? I guess the linux-anvil image has it, which is why this builds successfully. 

If there's no good way to get a conda package, then +1 on adding to extended base.

**Comment by bgruening** [comment]
@johanneskoester at least conda-forge is installing it via yum and following a similar strategy. I guess part of the reason are security updates etc.

I will take your comments as ok and proceed with merging the PRs?

**Comment by bgruening** [comment]
@daler I would just follow conda-forge here for the time being and not mess with openssh.
And yes all systems will need to have openssh-client installed. The linux-anvil has __not__ included openssh. Thats the reasons I added them in these PRs.

* https://github.com/bioconda/bioconda-utils/pull/211
* https://github.com/bioconda/bioconda-extended-base-image/pull/6

**Comment by daler** [comment]
Ah, I missed the bioconda-utils commit with the RUN line. And it makes sense to use the yum repos for security updates.

https://github.com/bioconda/bioconda-utils/pull/184 was an attempt to move toward a pre-built docker container which should be fine now that the build system is no longer changing frequently. If later we decide the build times get too long, moving to that pre-built model and adding openssh there would speed things up considerably.

**Comment by bgruening** [comment]
@planetmija thanks!

**Pull request merged**